README.md

```diff
@@ -251,10 +250,12 @@ The SDK will follow the same exact same lifecycle as with the 1st layer consent
 
 ## Understanding the `SPDelegate` protocol (delegate methods)
 
-### onSPUIReady(\_ controller: UIViewController
+### onSPUIReady(\_ controller: UIViewController)
 
 The SDK will wrap the web message into a `UIViewController` and call the `onSPUIReady` when there is a message to be displayed.
 
+**_NOTE:_** `onSPUIReady` overrides `modalPresentationStyle` of the ViewController to be `.overFullScreen`. This is done to avoid compatibility issues when ViewController can be swiped off the screen without any user action. We do not guarantee the stability of any other `modalPresentationStyle` except the `.overFullScreen` and highly encourage you to not change it.
```

Comment:
@Nevazhnovu I'd change a little bit the second port of the note like this:

```md
**_NOTE:_** `onSPUIReady` overrides `modalPresentationStyle` of the ViewController to be `.overFullScreen`. We identified some swiping gestures using other types of `modalPresentationStyle` causes the webview rendering the message to be unresponsive. We encourage you to keep using `.overFullScreen` until this issue is fixed.
```

Reply:
Looks good to me!